python/lsst/sims/maf/db/opsimDatabase.py

```diff
@@ -3,6 +3,7 @@
 import warnings
 from .Database import Database
 from lsst.sims.maf.utils import getDateVersion, TelescopeInfo
+from sqlalchemy import text
```

Comment:
yup, that's much better. I got confused by error messages and thought it didn't work there, but it turns out sqlalchemy just doesn't let you convert a text to a text.